src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype"),
+    url(./fonts/Avenir-Heavy.ttf) format("truetype"),
+    url(./fonts/Avenir.ttc) format("truetype");
+}
+
+body {
+  margin-top: 0;
+  font-family: Avenir, sans-serif;
+}
+
+.search {
+  display: flex;
+  flex-direction: column;
+}
+
+.search__bar {
+  margin-top: 20px;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.search__input {
+  border: none;
+  text-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  font-family: Avenir, sans-serif;
+  font-weight: 900;
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__bar:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+  outline-offset: none;
+}
+
+.search__bar--big {
+  height: 70px;
+}
+
+.search__icon--big {
+  position: absolute;
+  left: 25px;
+  top: 24px;
+  height: 19px;
+  width: 19px;
+}
+
+.search__input--big {
+  padding: 25px 0 26px 62px;
+  font-size: 16px;
+}
+
+.search__input--big::placeholder {
+  color: #3d4e61;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.search__bar--small {
+  height: 42px;
+}
```

Comment:
2'nd point in checklist, you don't need to create search__bar, search__bar--big and search__bar--small. Let's create base class search__bar with all needed styles, and override all necessary ones in search__bar--small